### PR TITLE
[doc] PR for Add YBA tab and info under YugabyteDB > Explorer > Set up your YugabyteDB cluster

### PR DESCRIPTION
**Pull request by lizayugabyte**
Addresses https://yugabyte.atlassian.net/browse/PLAT-6990

Preview: https://deploy-preview-15775--infallible-bardeen-164bc9.netlify.app/preview/explore/

### CODE REVIEW

**Comment by lizayugabyte** [comment]
Slack feedback from Amit Chauhan:

Hi Liza, looks all right to me. 
For this line:

_To view the application UI, navigate to http://localhost:8080/._

We should probably change it to http://<machine-ip-or-dns>:8080

**Review by akscjo — APPROVED** [review]

File: docs/content/preview/explore/_index.md [hunk 1]
```diff
@@ -15,57 +15,71 @@ type: indexpage
...
+- [Installed](../quick-start/linux/) YugabyteDB.
+- [Created an account](https://cloud.yugabyte.com/signup?utm_medium=direct&utm_source=docs&utm_campaign=Cloud_signup) in YugabyteDB Managed.
+- [Installed](../yugabyte-platform/install-yugabyte-platform/) YugabyteDB Anywhere and [configured](../yugabyte-platform/configure-yugabyte-platform/) it to run in AWS.
 
 <ul class="nav nav-tabs-alt nav-tabs-yb custom-tabs">
+  <li>
+    <a href="#local" class="nav-link active" id="local-tab" data-toggle="tab"
+      role="tab" aria-controls="local" aria-selected="true">
+      <img src="/icons/database.svg" alt="Server Icon">
+      Use a local universe
```
Comment (ddhodge) [inline docs/content/preview/explore/_index.md:57]: Local

File: docs/content/preview/explore/_index.md [hunk 2]
```diff
@@ -15,57 +15,71 @@ type: indexpage
...
+    </a>
+  </li>
   <li >
-    <a href="#cloud" class="nav-link active" id="cloud-tab" data-toggle="tab"
-      role="tab" aria-controls="cloud" aria-selected="true">
+    <a href="#cloud" class="nav-link" id="cloud-tab" data-toggle="tab"
+      role="tab" aria-controls="cloud" aria-selected="false">
       <img src="/icons/cloud.svg" alt="Cloud Icon">
-      Use a cloud cluster
+      Use a cloud universe
```
Comment (ddhodge) [inline docs/content/preview/explore/_index.md:64]: YugabyteDB Managed

File: docs/content/preview/explore/_index.md [hunk 3]
```diff
@@ -15,57 +15,71 @@ type: indexpage
...
   </li>
   <li>
-    <a href="#local" class="nav-link" id="local-tab" data-toggle="tab"
-      role="tab" aria-controls="local" aria-selected="false">
-      <img src="/icons/database.svg" alt="Server Icon">
-      Use a local cluster
+    <a href="#anywhere" class="nav-link" id="anywhere-tab" data-toggle="tab"
+      role="tab" aria-controls="anywhere" aria-selected="false">
+      <img src="/icons/server.svg" alt="Server Icon">
+      Use YugabyteDB Anywhere
```
Comment (ddhodge) [inline docs/content/preview/explore/_index.md:71]: YugabyteDB Anywhere

File: docs/content/preview/explore/_index.md [hunk 4]
```diff
@@ -250,27 +275,35 @@ Download the YB Workload Simulator JAR file using the following command:
...
 
-### Use the application
+## Use the application
 
 <ul class="nav nav-tabs-alt nav-tabs-yb custom-tabs">
+  <li>
+    <a href="#localworkload" class="nav-link active" id="local-tab" data-toggle="tab"
+      role="tab" aria-controls="local" aria-selected="true">
+      <img src="/icons/database.svg" alt="Server Icon">
+      Use a local universe
```
Comment (ddhodge) [inline docs/content/preview/explore/_index.md:285]: Same

**Review by ddhodge — APPROVED** [review]

File: docs/content/preview/explore/_index.md [hunk 5]
```diff
@@ -313,28 +346,103 @@ java -jar \
...
+         -Dspring.datasource.hikari.maximumPoolSize=100 \
+         -Dspring.datasource.hikari.data-source-properties.topologyKeys=<aws.regions.zones> \
+         -DloggingDir="/tmp/logs" \
+         -DXmx=32g \
+         -jar ./yb-workload-sim-0.0.2.jar
+   ```
+
+   Replace the following:
+
+   - `<node_ip>` - The IP address of the node in your YugabyteDB Anywhere universe. You can find this information by navigating to **Universes > UniverseName >Nodes** in YugabyteDB Anywhere.
```
[the remark this answers is not in the document]
Comment (lizayugabyte) [reply docs/content/preview/explore/_index.md:398]: fixed

File: docs/content/preview/explore/_index.md (hunk 1 above)
Comment (lizayugabyte) [reply docs/content/preview/explore/_index.md:57]: done

File: docs/content/preview/explore/_index.md (hunk 2 above)
Comment (lizayugabyte) [reply docs/content/preview/explore/_index.md:64]: done

File: docs/content/preview/explore/_index.md (hunk 3 above)
Comment (lizayugabyte) [reply docs/content/preview/explore/_index.md:71]: done

File: docs/content/preview/explore/_index.md (hunk 4 above)
Comment (lizayugabyte) [reply docs/content/preview/explore/_index.md:285]: done

File: docs/content/preview/explore/_index.md [hunk 6]
```diff
@@ -313,28 +346,103 @@ java -jar \
...
+
+   - `<port>` - 5433.
+
+   - `<aws.regions.zones>` - The zones in your universe, comma-separated, in the format `cloud.region.zone`, to be used as topology keys for [topology-aware load balancing](../drivers-orms/smart-drivers/#topology-aware-connection-load-balancing). Node details are displayed in **Universes > UniverseName >Nodes**. For example, to add topology keys for a single-region multi-zone universe, you would enter the following:
+
+     ```sh
+     -Dspring.datasource.hikari.data-source-properties.topologyKeys=aws.us-east-1.us-east-1a,aws.us-east-1.us-east-1b,aws.us-east-1.us-east-1c
+     ```
+
+The preceding instructions are applicable to a YSQL workload. To run a YCQL workload, add the following parameters before the `-jar ./yb-workload-sim-0.0.2.jar` command :
```
[the remark this answers is not in the document]
Comment (lizayugabyte) [reply docs/content/preview/explore/_index.md:412]: fixed

**Pull request merged**